### DISCUSSION TITLE
Быстрый поиск в истории

### DISCUSSION THREAD

**Discussion by kmserg1** [elfmz/far2l#2031] [Ideas]
Часто пользуюсь историей директорий. Полезная вещь, чтобы быстро переместитться в нужную папку.
Однако там хранятся сотни записей и если по последним записям можно легко перейти, то если надо 
найти что-то дальше, то это становится проблематичным.
Было-бы отлично, если бы в этом списке можно было быстро искать, подбно Alt+символы в файловой панели,
при этом хотелось бы, чтобы поиск был не по первым символам, а по любой части строки. 
Я думаю, что эту функцию можно было бы добавить во все списки.

**Comment by spnethw** [comment]
Ctrl-Alt-F  ?

**Reply by kmserg1** [reply]
Спасибо
